raster/src/main/scala/geotrellis/raster/VectorToRaster.scala

```diff
@@ -172,4 +172,11 @@ object VectorToRaster {
     }
     IntArrayTile(array, cols, rows)
   }
+/**
+ *Returns the output of countPoints function, since countPoints() is not a intuitive nomenclature
```

Comment:
Not good to have comments that refer to code that isn't there...that would be confusing to someone who had never seen the `countPoints` name.

I think `convertToRaster` sounds more like Rasterization, which is it's own set of code (which we have in the `rasterize` package). If `countPoints` isn't a good name, we should figure out a better one.

How about `cellCounts`? or `pointCountPerCell`?